### PR TITLE
Adding a new class brew and updating another class brew

### PR DESCRIPTION
**Pull request by Foxfire94**
Adding my new "Vampire" class to the repo and some minor adjustments to the Astartes Initiate.

### CODE REVIEW

File: class/Foxfire94; Vampire.json [hunk 1]
```diff
@@ -0,0 +1,11247 @@
...
+								"entries": [
+									"The creature is stunned for 1 minute. At the end of each of its turns it makes a Constitution saving throw against your power save DC, ending the stunned condition on a success."
+								]
+							}
+						],
+						[
+							{
+								"type": "cell",
+								"roll": {
+									"exact": "24+"
```
Comment (revilowaldow) [inline class/Foxfire94; Vampire.json]: This needs to be an integer.
`"exact": 24`

File: class/Foxfire94; Vampire.json [hunk 2]
```diff
@@ -0,0 +1,11247 @@
...
+				"s": true,
+				"m": "one point from your blood pool which is consumed by the power"
+			},
+			"duration": [
+				{
+					"type": "instant"
+				}
+			],
+			"entries": [
+				"Choose one creature you can see within range that is in bright or dim light. That creature must make a Strength saving throw against your power save DC as inky black tendrils attempt to pull them through their own shadow. On a failed save the creature disappears from its current location and instantly reappears in an unoccupied space
```
Comment (revilowaldow) [inline class/Foxfire94; Vampire.json]: Lowercase v
`"v": true`

Comment (revilowaldow) [inline class/Foxfire94; Vampire.json]: Lowercase v
`"v": true`

**Review by revilowaldow — CHANGES_REQUESTED** [review]
These three fixes should make it pass the schema.

File: class/Foxfire94; Vampire.json (hunk 2 above)
Comment (revilowaldow) [reply class/Foxfire94; Vampire.json]: Not sure what happened here, should be line 9887

Comment (revilowaldow) [reply class/Foxfire94; Vampire.json]: Not sure what happened here, should be line 9944

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**